### DISCUSSION TITLE
How to run two watch tasks for development?

### DISCUSSION THREAD

**Discussion by ziimakc** [vercel/turborepo#4986] [Help] [answered]
I'm a bit confused about how to run two tasks in watch mode for development.

1) Compile typescript and watch changes `tsc -w`
2) Run nodemon in watch mode

So I need to do something like `turbo run dev:emit dev:start`

Or am I forced to use parallel npm scripts? `turbo run dev:emit & turbo run dev:start`

```
{
  "extends": ["//"],
  "pipeline": {
    "dev:emit": {
      "cache": false,
      "persistent": true
    },
    "dev:start": {
      "cache": false,
      "persistent": true
    }
  }
}
```

**Comment by mehulkar** [comment, marked as answer]
`turbo run dev:emit dev:start` should just work?
